features/layout-builder-custom-blocks.feature

```diff
@@ -43,5 +43,36 @@ Feature: Layout Builder Custom Blocks Check
     Then I should see that the "#drupal-off-canvas > div > ul > li:nth-child(29) > a" element exists
     Then I should see the link "Webform"
 
-
-
+  @api @javascript @apparence_settings_test
```

Comment:
Tag has typo. Should be `@appearance_settings-test`

Reply:
Well my correction had a typo 🤣 too, shouldn't have been a `-` the issue was that `apparence` needed to be `appearance`.  I'll correct it. ;)